scripts/bert/finetune_classifier.py

```diff
@@ -90,93 +134,150 @@
 
 # model and loss
 dataset = 'book_corpus_wiki_en_uncased'
-bert, vocabulary = bert_12_768_12(dataset_name=dataset,
-                                  pretrained=True, ctx=ctx, use_pooler=True,
-                                  use_decoder=False, use_classifier=False)
-model = BERTClassifier(bert, dropout=0.1)
-model.classifier.initialize(init=mx.init.Normal(0.02), ctx=ctx)
-model.hybridize(static_alloc=True)
+bert, vocabulary = bert_12_768_12(
+    dataset_name=dataset,
+    pretrained=True,
+    ctx=ctx,
+    use_pooler=True,
+    use_decoder=False,
+    use_classifier=False)
+do_lower_case = 'uncased' in dataset
+tokenizer = FullTokenizer(vocabulary, do_lower_case=do_lower_case)
 
-loss_function = gluon.loss.SoftmaxCELoss()
+if task_name in ['STS-B']:
+    model = BERTRegression(bert, dropout=0.1)
+    model.regression.initialize(init=mx.init.Normal(0.02), ctx=ctx)
+    loss_function = gluon.loss.L2Loss()
+else:
+    model = BERTClassifier(bert, dropout=0.1)
```

Comment:
`num_classes` should be passed in explicitly, e.g. MNLI has 3 classes.

Reply:
`MNLIDataset` class contains `num_classes` information. Because GLUE tasks were already defined on paper.